scalableminds/webknossos
# Fix rerender after each layer name keypress in dataset import view

### URL of deployed dev instance (used for testing):
- https://editviewfixrerender.webknossos.xyz (I’ll deploy after CI success)

### Steps to test:
- Import a remote zarr dataset
- In the edit view, change a layer name
- typing there should not trigger re-render after each keypress, causing the text field to lose focus

### Issues:
- fixes #6624

------
